run-build.sh

```diff
@@ -61,51 +63,44 @@ CUSTOM_BUILD_ARGS=
 # Set nuget package cache under the repo
 [ -z $NUGET_PACKAGES ] && export NUGET_PACKAGES="$REPOROOT/.nuget/packages"
 
-args=( "$@" )
+args=( )
 
 while [[ $# > 0 ]]; do
     lowerI="$(echo $1 | awk '{print tolower($0)}')"
     case $lowerI in
         -c|--configuration)
             export CONFIGURATION=$2
-            args=( "${args[@]/$1}" )
-            args=( "${args[@]/$2}" )
             shift
             ;;
         --nopackage)
             export DOTNET_BUILD_SKIP_PACKAGING=1
-            args=( "${args[@]/$1}" )
             ;;
         --skip-prereqs)
             # Allow CI to disable prereqs check since the CI has the pre-reqs but not ldconfig it seems
             export DOTNET_INSTALL_SKIP_PREREQS=1
-            args=( "${args[@]/$1}" )
             ;;
         --nobuild)
             BUILD=0
             ;;
         --architecture)
             ARCHITECTURE=$2
-            args=( "${args[@]/$1}" )
-            args=( "${args[@]/$2}" )
             shift
             ;;
         --runtime-id)
             CUSTOM_BUILD_ARGS="/p:Rid=\"$2\""
-            args=( "${args[@]/$1}" )
-            args=( "${args[@]/$2}" )
             shift
             ;;
         # This is here just to eat away this parameter because CI still passes this in.
         --targets)            
-            args=( "${args[@]/$1}" )
-            args=( "${args[@]/$2}" )
             shift
             ;;
         --linux-portable)
             LINUX_PORTABLE_INSTALL_ARGS="--runtime-id linux-x64"
             CUSTOM_BUILD_ARGS="/p:Rid=\"linux-x64\" /p:OSName=\"linux\" /p:IslinuxPortable=\"true\""
-            args=( "${args[@]/$1}" )
+            ;;
+        --bootstrap-cli)
```

Comment:
Can this get added to the `--help` below?

Reply:
I think the name should match the terminology already used in the repo - `stage0`.

---
In reply to: [140321620](https://github.com/dotnet/cli/pull/7688#discussion_r140321620) [](ancestors = 140321620)